app/src/main/res/values/colors_migrating.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <!-- contains color declarations which are supposed to be in [color_defs.xml] but has not been
+   renamed yet to have actual color name instead of names linked to their use and components.
+   This is a temporary measure to make sure other 4 color files follows the convention decided
+   for them. -->
```

Comment:
```suggestion
  <!-- Color declarations which are supposed to be in color_defs.xml but have not yet been
   renamed to have actual color name instead of names linked to their use and components.
   This is a temporary measure to make sure other 4 color files follows the convention decided
   for them. -->
```

Some fixes in the beginning, but content wise: maybe it would be clearer just to say that all of these colors will be moved to, or replaced with one in, a color in one of the other three files & what those files are.

The temporary sentence at the end sounds good.

Reply:
Please also add a TODO pointing to an issue to remove this file once the migration is complete.

Reply:
Done.